go.mod

```diff
@@ -3,18 +3,22 @@ module github.com/step-security/secure-workflows
 go 1.17
 
 require (
-	github.com/aws/aws-lambda-go v1.25.0
-	github.com/aws/aws-sdk-go v1.40.23
+	github.com/aws/aws-lambda-go v1.28.0
+	github.com/aws/aws-sdk-go v1.42.38
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
 require (
-	github.com/golang/protobuf v1.4.2 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
```

Comment:
Don't update dependencies as part of this change. We can update them as part of separate PR (dependabot is setup for this)